metrics/cli/lib/interfaces/cli/cli.dart

```diff
@@ -1,8 +1,32 @@
 // Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
+import 'dart:async';
+import 'dart:io';
+
+import 'package:process_run/process_run.dart' as cmd;
+
 /// A base class for CLIs that provides common methods for them.
 abstract class Cli {
+  /// An executable name of this CLI.
+  String get name;
+
   /// Shows the version information of this CLI.
   Future<void> version();
+
+  /// Runs the executable by the [name].
```

Comment:
Would it be great to extend this documentation with the default parameters and maybe a short description of the parameters and the behavior when the parameters are null? For example, I believe it would be great to mention that if the `workingDirectory` is `null`, the command will run in the application directory.